Troupon.Catalog.Api/Controllers/SimulationController.cs

```diff
@@ -23,30 +23,30 @@ public SimulationController(IMapper mapper, IMediator mediator)
     {
     }
 
-    [ApiExplorerSettings(IgnoreApi = true)]
-    [HttpPost("HandleDealEventPublished")]
-    public async Task<ActionResult<DealDto>> Post([FromBody] UpdateDealViewCommand model, CancellationToken cancellationToken)
-    {
-      try
-      {
-        var result = await Mediator.Send(model, cancellationToken);
+    // [ApiExplorerSettings(IgnoreApi = true)]
```

Comment:
Please remove commented out code. If we need to bring it back, it's in the git repository.

Reply:
Done!